pytential/symbolic/pde/scalar.py

```diff
@@ -400,8 +400,10 @@ def Sp(density):
             if isinstance(self.kernel, HelmholtzKernel):
                 DpS0u = (
                         sym.Dp(
-                            self.kernel - self.laplace_kernel,
-                            laplace_s_inv_sqrt_w_u,
+                            self.kernel, laplace_s_inv_sqrt_w_u,
+                            qbx_forced_limit=+1, **kwargs)
+                        - sym.Dp(
+                            self.laplace_kernel, laplace_s_inv_sqrt_w_u,
```

Comment:
This seems to have been broken for quite a while!

Reply:
You mean the difference kernels no longer work? Yikes. Should have had tests for them. I guess it makes sense that FMMs for them don't work directly (no PDE after all), but for the P2P, there's no real reason to not support them.

@isuruf What do you think?

Also, this is maybe a separate discussion from this PR.

Reply:
Yes, I meant the difference between `sumpy` kernels in there. I was the last one who touched this code, but it seems to have been this way for a while, so not sure when it broke.

Agreed this is for a different PR, just wanted to make a note of it before I forgot.

Reply:
#156